src/dev/i18n/extract_code_messages.test.js

```diff
@@ -81,11 +84,13 @@ describe('extractCodeMessages', () => {
 
 describe('isIntlFormatMessageFunction', () => {
   test('detects intl.formatMessage call expression', () => {
-    const callExpressioNode = [...traverseNodes(parse(intlFormatMessageSource).program.body)].find(
+    const callExpressioNodes = [...traverseNodes(parse(intlFormatMessageSource).program.body)].filter(
       node => isCallExpression(node)
     );
 
-    expect(isIntlFormatMessageFunction(callExpressioNode)).toBe(true);
+    callExpressioNodes.forEach((callExpressioNode) => {
```

Comment:
__issue:__ try not to do `expect` in the loop, there is a chance to introduce bug that will make `callExpressionNodes` an empty array and this test will pass, I'd rather do something like this instead:
```js
expect(callExpressionNodes).toHaveLength(4);
expect(callExpressionNodes.every(callExpressionNode => isIntlFormatMessageFunction(callExpressionNode))).toBe(true);
```

Reply:
good point, done, thanks